src/factories/ModuleFactory.sol

```diff
@@ -125,6 +126,18 @@ contract ModuleFactory is IModuleFactory, Ownable2Step {
         return (_beacons[id], id);
     }
 
+    function supportsInterface(bytes4 interfaceId)
+        public
+        view
+        virtual
+        override(ERC165)
+        returns (bool)
+    {
+        bytes4 interfaceId_IModuleFactory = type(IModuleFactory).interfaceId;
```

Comment:
Cant you just do:
return interfaceId == type(IModuleFactory).interfaceId || ....